scripts/sql/108_create_semgrep_plugin.up.sql

```diff
@@ -0,0 +1,87 @@
+INSERT INTO "plugin_metadata" ("id", "name", "description","type","icon","deleted", "created_on", "created_by", "updated_on", "updated_by") VALUES (nextval('id_seq_plugin_metadata'), 'Semgrep','Semgrep is a fast, open source, static analysis engine for finding bugs, detecting dependency vulnerabilities, and enforcing code standards.','PRESET','https://raw.githubusercontent.com/devtron-labs/devtron/main/assets/semgrep.png','f', 'now()', 1, 'now()', 1);
+
+INSERT INTO "plugin_tag_relation" ("id", "tag_id", "plugin_id", "created_on", "created_by", "updated_on", "updated_by") VALUES (nextval('id_seq_plugin_tag_relation'), 2, (SELECT id FROM plugin_metadata WHERE name='Semgrep'),'now()', 1, 'now()', 1);
+INSERT INTO "plugin_tag_relation" ("id", "tag_id", "plugin_id", "created_on", "created_by", "updated_on", "updated_by") VALUES (nextval('id_seq_plugin_tag_relation'), 3, (SELECT id FROM plugin_metadata WHERE name='Semgrep'),'now()', 1, 'now()', 1);
+
+INSERT INTO "plugin_pipeline_script" ("id", "script","type","deleted","created_on", "created_by", "updated_on", "updated_by")
+VALUES (
+    nextval('id_seq_plugin_pipeline_script'),
+    '#!/bin/sh
+set -eo pipefail
+chmod 741 /devtroncd
+chmod 741 /devtroncd/*
+apk add py3-pip
+pip install pip==21.3.1
+pip install semgrep
+export SEMGREP_APP_TOKEN=$SemgrepAppToken
+SemgrepTokenLen=$(echo -n $SEMGREP_APP_TOKEN | wc -m)
+if [ $((SemgrepTokenLen)) == 0 ]
+then
+    SEMGREP_APP_TOKEN=$SEMGREP_API_TOKEN
```

Comment:
we need to export this token again

Reply:
it is already exported and we are assigning value to already exported token.